datasets/flwr_datasets/partitioner/continuous_partitioner.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2025 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Continuous partitioner class that works with Hugging Face Datasets."""
+
+
+# pylint: disable=R0913, R0917
+from typing import Optional
+
+import numpy as np
+
+from datasets import Dataset
+from flwr_datasets.partitioner import Partitioner
```

Comment:
```suggestion
from flwr_datasets.partitioner.partitioner import Partitioner
```

Reply:
Note you also need to add line:
```python
from .continuous_partitioner import ContinuousPartitioner
```
at the top of the `__init__.py` in the partitioners directory and then add `"ContinuousPartitioner"` to the `__all__` list.